docs/source/upgrading/search_migration.rst

```diff
@@ -0,0 +1,27 @@
+Porting A Search Query
+======================
+
+Here is an example of executing a PIF search query against Citrination:
+
+.. code-block:: python
+
+    query_dataset = PifSystemReturningQuery(size=5, 
+                                            query=DataQuery(
+                                                dataset=DatasetQuery(
+                                                    id=Filter(equal=str(my_dataset_id))
+                                             )))
+    query_result = client.pif_search(query_dataset)
+
+The only difference required in the new version of the client is to use make sure you are calling the ``pif_search`` method on the ``search`` client.
```

Comment:
The "Attention" block suggests this isn't actually required.

Reply:
👍